rasa/core/actions/forms.py

```diff
@@ -584,6 +585,28 @@ async def activate(
         # collect values of required slots filled before activation
         prefilled_slots = {}
 
+        # run slot extraction the second time during form activation
+        # for slots with mapping conditions
```

Comment:
should we update the docstring of the function?

Reply:
Yup, I'll see how I can add more context to the docstring about this new code.

Reply:
Done: https://github.com/RasaHQ/rasa/pull/11149/commits/8c87ff1eac90aca80a73d27c3d56afbb8e9bae74